tests/ui/lifetimes/elided-lifetimes-in-type-paths.rs

```diff
@@ -0,0 +1,57 @@
+#![deny(elided_lifetimes_in_paths)]
+
+// Most of the time, we focus on elided lifetimes in function
+// signatures, but they can also appear in other places! The original
+// version of this lint handled all these cases in one location, but
+// it's desired that the one lint actually be multiple.
```

Comment:
Nit (wording): maybe for clarity (I was reading this, and it wasn't immediately clear)

> Most of the time, we focus on elided lifetimes in function signatures, but they can also appear in other places! The original lint `elided_lifetime_in_paths` handled all these cases in one location, but it's more desirable to split it into 3 more fine-grained lints under a new `hidden_lifetimes_in_paths` lint group umbrella:
>
> 1. `hidden_lifetimes_in_input_paths`
> 2. `hidden_lifetimes_in_output_paths`
> 3. `hidden_lifetimes_in_type_paths`